frontend/src/ui/time.tsx

```diff
@@ -30,23 +36,39 @@ export const RelativeDate: React.FC<RelativeDateProps> = ({ date }) => {
     const MONTH = 30.5 * DAY;
     const YEAR = 365.25 * DAY;
 
+    const liveDate = ((time: number, unit: Intl.RelativeTimeFormatUnit) => {
+        const parts = intl.formatToParts(time, unit);
+        return String(parts[1].value) + parts[2].value;
```

Comment:
Unfortunately, I don't think we can use this. The docs for `formatToParts` don't promise a whole lot, certainly not that the second and third element in the array are the one we are interested. (Other languages might have different word order.) And I don't see any method that would reliably give us the translations for "unit". 

So I fear we have to add translations for all units we are interested in. It's not too bad though. Maybe add `general.time` and add a bunch of translations there?

Reply:
Oh, it's probably worse since we have to add plural and singular form. i18n has some special syntax for that I think. So each translation should take an argument, i.e. `seconds: {{value}} Sekunden`. But yeah, I don't see another way unfortunately :/

Reply:
If the text doesn't need to say "Live in..." it might be possible to reuse the `RelativeTimeFormat` string like this:
<img width="468" alt="Bildschirmfoto 2022-12-01 um 18 03 21" src="https://user-images.githubusercontent.com/94838646/205264221-dba71d8e-d813-4abb-9eb1-6098382782f7.png">
<img width="468" alt="Bildschirmfoto 2022-12-01 um 18 02 52" src="https://user-images.githubusercontent.com/94838646/205264185-1023be8f-56d7-4b62-8d22-06f027327bd0.png">
I believe this should work for most, if not all languages - but of course I can't say for sure.
A similar construct is already used in the placeholder for live events: https://github.com/elan-ev/tobira/blob/0359621bd00d56251ab4b04ca7ca8d866740103d/frontend/src/ui/player/index.tsx#L249

Reply:
That's clever!